policy/modules/system/fstools.te

```diff
@@ -176,6 +181,10 @@ ifdef(`distro_debian',`
 
 ifdef(`distro_gentoo',`
```

Comment:
Up to @pebenito if this should be in the gentoo-specific block. I don't imagine this would be much different on other distros?